src/main/target/ALIENWHOOP/config.c

```diff
@@ -59,27 +65,84 @@
 
 void targetConfiguration(void)
 {
-    if (hardwareMotorType == MOTOR_BRUSHED) {
+    if (hardwareMotorType == MOTOR_BRUSHED) 
+    {
         motorConfigMutable()->dev.motorPwmRate = BRUSHED_MOTORS_PWM_RATE;
-    motorConfigMutable()->minthrottle = 1080;
-    motorConfigMutable()->maxthrottle = 2000;
-        pidConfigMutable()->pid_process_denom = 1;
+        motorConfigMutable()->minthrottle = 1080;
+        motorConfigMutable()->maxthrottle = 2000;
     }
 
-    rxConfigMutable()->serialrx_provider = SERIALRX_SBUS;
+    /* Default to Spektrum */
+    rxConfigMutable()->serialrx_provider = SERIALRX_SPEKTRUM2048;
+    rxConfigMutable()->spektrum_sat_bind = 5;
+    rxConfigMutable()->spektrum_sat_bind_autoreset = 1;
+    parseRcChannels("TAER1234", rxConfigMutable());
 #if defined(ALIENWHOOPF4)
     rxConfigMutable()->sbus_inversion = 0; // TODO: what to do about F4 inversion?
 #else
     rxConfigMutable()->sbus_inversion = 1; // invert on F7
 #endif
 
-/* Breadboard-specific settings for development purposes only
- */
+    beeperOffSet((BEEPER_BAT_CRIT_LOW | BEEPER_BAT_LOW | BEEPER_RX_SET) ^ BEEPER_GYRO_CALIBRATED);
+
+    /* Breadboard-specific settings for development purposes only
+     */
 #if defined(BREADBOARD)
     boardAlignmentMutable()->pitchDegrees = 90; // vertical breakout board
     barometerConfigMutable()->baro_hardware = BARO_DEFAULT; // still testing not on V1 or V2 pcb
+#else
+    barometerConfigMutable()->baro_hardware = BARO_NONE;
 #endif
 
     compassConfigMutable()->mag_hardware =  MAG_DEFAULT;
+
+    /* F4 (especially overclocked) and F7 ALIENWHOOP perform splendidly with 32kHz gyro enabled */
+    gyroConfigMutable()->gyro_use_32khz = 1;
+    gyroConfigMutable()->gyro_sync_denom = 2;  // 16kHz gyro
+    pidConfigMutable()->pid_process_denom = 1; // 16kHz PID
+
+    featureSet((FEATURE_DYNAMIC_FILTER | FEATURE_AIRMODE | FEATURE_ANTI_GRAVITY) ^ FEATURE_RX_PARALLEL_PWM);
+
+    /* AlienWhoop PIDs based on Ole Gravy Leg (aka Matt Williamson's) PIDs 
+     */
+    for (int profileId = 0; profileId < MAX_PROFILE_COUNT; profileId++) 
```

Comment:
Can you change to K&R style, ie `for (int profileId = 0; profileId < MAX_PROFILE_COUNT; profileId++)  {`

Reply:
Yes no problem. 😄